libs/core/src/lib/select/select.component.ts

```diff
@@ -70,6 +70,14 @@ export class SelectComponent implements OnChanges, AfterContentInit, OnDestroy,
     @Input()
     maxHeight: string;
 
+    /** Select type defines the border type of the select button. */
+    @Input()
+    selectType: string = '';
```

Comment:
yes, the default border-radius is given by core which is square, which we cannot override if we remove it might affect the look of core select. so i have added a separate class to override the css.

i have used regular-border class to allow default select type with border-radius of 4px , if we don't use then what ever the default defined in the fd-select-button-custom gets loaded. button class border-radius is not loaded.